.internal-ci/helm/fog-services/templates/fog-report-grpc-ingress.yaml

```diff
@@ -13,6 +13,9 @@ metadata:
     {{- if $.Values.global.certManagerClusterIssuer }}
     cert-manager.io/cluster-issuer: {{ $.Values.global.certManagerClusterIssuer }}
     {{- end }}
+    {{- if (include "fogServices.blacklist.enabled" $) }}
+    haproxy.org/blacklist: {{ include "fogServices.blacklist.pattern" $ }}
```

Comment:
:question: Does the `$` change behaviour versus the other changes that use `.`?
The other locations use `include "blah" .`
Looking at https://helm.sh/docs/howto/charts_tips_and_tricks/#using-the-include-function I'm not seeing information on `$` vs `.`

Reply:
Because this is a loop, you need to use `$` to access the top level scope. `.` would only access the inner loop scope.